internal/manifests/collector/ports.go

```diff
@@ -21,19 +21,22 @@ import (
 )
 
 const (
-	StatsD          = "statsd"
-	CollectD        = "collectd"
-	XrayProxy       = "aws-proxy"
-	XrayTraces      = "aws-traces"
-	OtlpGrpc        = "otlp-grpc"
-	OtlpHttp        = "otlp-http"
-	AppSignalsGrpc  = "appsignals-grpc"
-	AppSignalsHttp  = "appsignals-http"
-	AppSignalsProxy = "appsignals-xray"
-	EMF             = "emf"
-	EMFTcp          = "emf-tcp"
-	EMFUdp          = "emf-udp"
-	CWA             = "cwa-"
+	StatsD            = "statsd"
+	CollectD          = "collectd"
+	XrayProxy         = "aws-proxy"
+	XrayTraces        = "aws-traces"
+	OtlpGrpc          = "otlp-grpc"
+	OtlpHttp          = "otlp-http"
+	AppSignalsGrpc    = "appsignals-grpc"
```

Comment:
Seems like the operator code still contains a mix of `appsignals` and `applicationsignals`